week-1/exercises/initials.js

```diff
@@ -4,3 +4,10 @@ let lastName = "Johnson";
 
 // Declare a variable called initials that stores the first 3 characters of each string in upper case
 // Log the variable in each case
+
+let initials = firstName.slice(0, 3).toUpperCase();
+console.log(initials);
+let initials = middleName.slice(0, 3).toUpperCase();
+console.log(initials);
+let initials = lastName.slice(0, 3).toUpperCase();
+console.log(initials);
```

Comment:
You should create a variable that holds all three substrings in one variable. 

You only need to `let` a variable once, on its initial declaration